api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +35,12 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
-	
+
+	private Set<Integer> members;
```

Comment:
The type for the items of the items in this Set should be `CohortMember` and not `Integer`